pipenv/vendor/README.md

```diff
@@ -6,15 +6,21 @@ you need to patch anything, move the package to `patched` and generate a
 patch for it using `git diff -p <dependency_root_dir>`. This patch belongs
 in `./pipenv/tasks/vendoring/patches/patched/<packagename.patchdesc>.patch`.
 
-To add a vendored dependency or to update a single dependency, use the
-vendoring scripts:
+To add a vendored dependency or to update a single dependency, add the package
+name and version to `pipenv/vendor/vendor.txt`, for example:
+
+```
+appdirs==1.4.4
+```
+
+And the run the vendoring script:
+
 ```
-    pipenv run inv vendoring.update --package="pkgname==versionnum"
+python -m invoke vendoring.update
```

Comment:
We should ensure invoke is installed in the current environment.

`pipenv run inv` should work if you installed with `pipenv install -d`